src/main/java/mate/academy/spring/dao/impl/TicketDaoImpl.java

```diff
@@ -11,4 +13,11 @@ public class TicketDaoImpl extends AbstractDao<Ticket> implements TicketDao {
     public TicketDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
+
+    @Override
+    public Optional<Ticket> get(Long id) {
```

Comment:
I can't see usage of this method added

Reply:
![image](https://user-images.githubusercontent.com/118556810/224702022-2c6064fd-25cf-4aeb-a463-1e1b5136d1aa.png)